internal/gojs/builtin.go

```diff
@@ -1,18 +1,24 @@
 package gojs
 
 import (
-	"net/http"
-
-	"github.com/tetratelabs/wazero/internal/gojs/custom"
+	"github.com/tetratelabs/wazero/internal/gojs/config"
 	"github.com/tetratelabs/wazero/internal/gojs/goos"
 )
 
 // newJsGlobal = js.Global() // js.go init
-func newJsGlobal(rt http.RoundTripper) *jsVal {
+func newJsGlobal(config *config.Config) *jsVal {
 	var fetchProperty interface{} = goos.Undefined
-	if rt != nil {
+	uid, gid, euid := config.Uid, config.Gid, config.Euid
+	// We don't populate groups because our stat can't read them back
```

Comment:
even when we don't, TestChown tries to verify the stat has the correct gid on files afterwards. So, it still fails. I'll look into if we can return the group info or not.. possibly backfilling with -1 if that's what windows does.